src/renderers/shared/fiber/ReactFiberStack.js

```diff
@@ -34,6 +34,21 @@ exports.createCursor = function<T>(defaultValue : T) : StackCursor<T> {
   };
 };
 
+exports.getPrevious = function<T>(
+  cursor : StackCursor<T>,
+) : T | null {
+  if (index < 0) {
+    if (__DEV__) {
+      warning(false, 'Unexpected read of previous.');
+    }
+    return null;
+  }
+  if (index === 0) {
```

Comment:
Rather than "unnecessary" I should have said "bad" because it would prevent the _default_ value from being returned as previous if one was set. (eg like the default for `didPerformWorkStackCursor`)